agent/engine/task_manager.go

```diff
@@ -606,6 +611,10 @@ func (mtask *managedTask) cleanupTask(taskStoppedDuration time.Duration) {
 	mtask.engine.sweepTask(mtask.Task)
 
 	// TODO: Remove platform task resources
+	err := mtask.CleanupPlatformResources()
+	if err != nil {
+		seelog.Warnf("Unable to cleanup platform resources: %v", err)
```

Comment:
Can you change `platform resource` to be more specific? `platform resource` isn't very clear to me.

Reply:
I don't know that we can. There isn't a windows equivalent to this, but we need a generic hook here. I think we are all open to suggestions though 😄

Reply:
I'm considering cgroups to be a platform specific resource at the task level.
Everything worked upon here has that idea. 

I'd like to hear of alternatives that you think would make this clearer.